health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -70,26 +70,27 @@ export const UICustomizations = {
       const campaignName = searchParams.get("name");
       const projectType = searchParams.get("projectType");
       const campaignId = searchParams.get("campaignId");
-      switch (key) {
-       
+      switch (key) {       
           case "STATUS":
             
-            // const [localIsActive, setLocalIsActive] = useState(rowDataCache[serviceCode].isActive);
+            const [localIsActive, setLocalIsActive] = useState(row?.ServiceRequest?.[0]?.isActive);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)